docs/modeling/compound-models.rst

```diff
@@ -100,8 +100,9 @@ Some terminology
 
 It is possible to create new models just by
 combining existing models using the arithmetic operators ``+``, ``-``, ``*``,
-``/``, and ``**``, as well as by model composition using ``|`` and
-concatenation (explained below) with ``&``.
+``/``, and ``**``, or by model composition using ``|`` and
+concatenation (explained below) with ``&``, as well as using the :func:`~astropy.modeling.fix_inputs`
+and :func:`~astropy.convolution.convolve_models` functions.
```

Comment:
It would be useful to add descriptions of fix_inputs and convolve_models to the "Comprehensive description" section.

Reply:
I removed `convolve_models` because it's not in the `convolution` docs and added a link to the `fix_inputs` description.